local-test-infra/fsdiff-pod.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: node-master
+    test-network-function.com/generic: fs_diff_master
+    test-network-function.com/container: fs_diff_master
+  name: node-master
+  namespace: {{ NAMESPACE_TO_GENERATE }}
+spec:
+  # hostPID: true
+  containers:
+    - command:
+        - tail
+        - -f
+        - /dev/null
+      image: quay.io/isaacdorfman/centos-podman
```

Comment:
should this image be moved to quay.io/testnetworkfunction?

Reply:
Yes it should. Who has the write right for that repo ?